src/less/badge/ds4/badge.less

```diff
@@ -0,0 +1,12 @@
+@import '../base/badge.less';
+@import "../../less/ds4/variables.less";
+@import "../../less/ds4/mixins.less";
+
+.badge {
+    background: @ds4-color-core-red;
+    color: @ds4-color-core-white;
+
+    &-container .badge {
+        border: 2px solid @ds4-color-core-white;
```

Comment:
Let's add `border: 2px solid;` to the `/badge/base/badge.less` file, and just do `border-color: @ds4-color-core-white;` here. That makes it explicit we're only setting the color here, and not the entire border rule.